ongaku/client.py

```diff
@@ -181,6 +181,48 @@ def from_tanjun(
 
         return cls
 
+    @classmethod
+    def from_lightbulb(
+        cls,
+        client: lightbulb.GatewayEnabledClient,
+        *,
+        session_handler: typing.Type[SessionHandler] = BasicSessionHandler,
+        logs: str | int = "INFO",
+        attempts: int = 3,
+    ) -> Client:
+        """From Lightbulb.
+
+        This supports `client` and `player` [injection](../gs/injection.md) for [Lightbulb](https://github.com/tandemdude/hikari-lightbulb)
+
+        Example
+        -------
+        ```py
+        bot = hikari.GatewayBot(...)
+        client = lightbulb.Client(bot)
+        ongaku_client = ongaku.Client.from_arc(client)

```

Comment:
```suggestion
        ongaku_client = ongaku.Client.from_lightbulb(client)

```

Reply:
I need to recieve events like `VoiceStateUpdate`, `VoiceServerUpdate` and the ability to update the voice state. As long as I can do those, idm changing stuff.